doc/references.bib

```diff
@@ -1,102 +1,138 @@
-  pages = {1178-1193},
+  author     = {Stam, Cornelis J. and Nolte, Guido and Daffertshofer, Andreas},
+  doi        = {10.1002/hbm.20346},
+  journal    = {Human Brain Mapping},
+  number     = {11},
+  pages      = {1178-1193},
   shorttitle = {Phase Lag Index},
-  title = {Phase Lag Index: Assessment of Functional Connectivity from Multi Channel {{EEG}} and {{MEG}} with Diminished Bias from Common Sources},
-  volume = {28},
-  year = {2007}
+  title      = {Phase Lag Index: Assessment of Functional Connectivity from Multi Channel {{EEG}} and {{MEG}} with Diminished Bias from Common Sources},
+  volume     = {28},
+  year       = {2007}
 }
 
 @article{VinckEtAl2010,
-  author = {Vinck, Martin and {van Wingerden}, Marijn and Womelsdorf, Thilo and Fries, Pascal and Pennartz, Cyriel M.A.},
-  doi = {10.1016/j.neuroimage.2010.01.073},
-  journal = {NeuroImage},
-  number = {1},
-  pages = {112-122},
+  author     = {Vinck, Martin and {van Wingerden}, Marijn and Womelsdorf, Thilo and Fries, Pascal and Pennartz, Cyriel M.A.},
+  doi        = {10.1016/j.neuroimage.2010.01.073},
+  journal    = {NeuroImage},
+  number     = {1},
+  pages      = {112-122},
   shorttitle = {The Pairwise Phase Consistency},
-  title = {The Pairwise Phase Consistency: A Bias-Free Measure of Rhythmic Neuronal Synchronization},
-  volume = {51},
-  year = {2010}
+  title      = {The Pairwise Phase Consistency: A Bias-Free Measure of Rhythmic Neuronal Synchronization},
+  volume     = {51},
+  year       = {2010}
 }
 
 @article{BrunaEtAl2018,
-  doi = {10.1088/1741-2552/aacfe4},
-  year = {2018},
+  doi       = {10.1088/1741-2552/aacfe4},
+  year      = {2018},
   publisher = {{IOP} Publishing},
-  volume = {15},
-  number = {5},
-  pages = {056011},
-  author = {Ricardo Bru{\~{n}}a, Fernando Maest{\'{u}}, Ernesto Pereda},
-  title = {Phase locking value revisited: teaching new tricks to an old dog},
-  journal = {Journal of Neural Engineering},
+  volume    = {15},
+  number    = {5},
+  pages     = {056011},
+  author    = {Ricardo Bru{\~{n}}a, Fernando Maest{\'{u}}, Ernesto Pereda},
+  title     = {Phase locking value revisited: teaching new tricks to an old dog},
+  journal   = {Journal of Neural Engineering}
 }
 
 @article{NolteEtAl2004,
-  author = {Nolte, Guido and Bai, Ou and Wheaton, Lewis and Mari, Zoltan and Vorbach, Sherry and Hallett, Mark},
-  doi = {10.1016/j.clinph.2004.04.029},
+  author  = {Nolte, Guido and Bai, Ou and Wheaton, Lewis and Mari, Zoltan and Vorbach, Sherry and Hallett, Mark},
+  doi     = {10.1016/j.clinph.2004.04.029},
   journal = {Clinical Neurophysiology},
-  number = {10},
-  pages = {2292-2307},
-  title = {Identifying True Brain Interaction from {{EEG}} Data Using the Imaginary Part of Coherency},
-  volume = {115},
-  year = {2004}
+  number  = {10},
+  pages   = {2292-2307},
+  title   = {Identifying True Brain Interaction from {{EEG}} Data Using the Imaginary Part of Coherency},
+  volume  = {115},
+  year    = {2004}
```

Comment:
I know that technically it doesn't matter, but I would advise against all these whitespace changes to the BibTeX entries.  Keeping the formatting of this file similar to the MNE-Python BibTeX files seems preferable, in case references are shared between them.

Similarly, there is a consistent key naming scheme used in MNE-Python that is diverged from here in keys `kilian_2006` and `li_linear_2017`.

If you decide to keep the whitespace changes, I would advise also (1) updating Li et al 2017 to also have it, and (2) updating Hosking, Kilian, and Li et al to have alphabetically-ordered keys like the rest of the entries do.

Reply:
+1 for keeping the formatting similar to MNE-Python